src/generate.js

```diff
@@ -338,8 +338,24 @@ module.exports = {
 
         // copy local defined models to dist directory keeping the local folder structure.
         task('copy-ts-model', ['cleanup-dist', 'read-rest-api'], function () {
-           return gulp.src([path.join(source, 'ts') + '/**/*.ts'], {base: source})
-                .pipe(gulp.dest('model', {cwd: dest}));
+            let sourcePaths;
+            let sourceOptions;
+            let destination;
+            const defaultPath = path.join(source, 'ts');
```

Comment:
Should we define the temporary `defaultPath` only in the block where it actually is needed? It looks as only the 1st if branch refers it.

Reply:
true...

Reply:
don't agree, we check it's existence before entering the block itself.

Reply:
Ah :) That's a good point. I missed that 👍